src/Extension/CoreExtension.php

```diff
@@ -1199,7 +1199,7 @@ public static function length(string $charset, $thing): int
             return mb_strlen($thing, $charset);
         }
 
-        if (is_countable($thing) || $thing instanceof \SimpleXMLElement) {
```

Comment:
SimpleXMLElement is countable.

Reply:
Since 8.0. As we bumped the min version, this change is valid.